src/main/java/de/retest/web/selenium/TestHealer.java

```diff
@@ -152,10 +141,32 @@ private WebElement findElementByLinkText( final ByLinkText by ) {
 	}
 
 	private WebElement findElementByCssSelector( final ByCssSelector by ) {
-		final String selector = retrieveUsableCssSelector( by );
-
-		final Element actualElement = de.retest.web.selenium.By.findElementByAttribute( lastExpectedState,
-				lastActualState, CLASS, value -> ((String) value).contains( selector ) );
+		final String selector = ByWhisperer.retrieveCssSelector( by );
+		if ( selector.matches( ".*[<>:+\\s\\*].*" ) ) {
+			throw new IllegalArgumentException(
+					"For now, only simple CSS selectors are implemented. Please report your chosen selector ('"
+							+ selector + "') at https://github.com/retest/recheck-web/issues." );
```

Comment:
Ah. Actually I wanted to point users to create a separate issue for any things that come up. So this URL was intentional.

Reply:
Well, any issues that arise through the exception are related to the mentioned issues (which did not exist at the time writing these messages). Those should track the progress, we do not want duplicate issues for unsupported css selectors.

Reply:
Hmm. I feel differently: We should have one issue per yet unsupported CSS selector, so we can close the ones that are supported. This would mean smaller steps and finer tracking...

Reply:
Do you know tasks? We can update this issue with one task for each selector reported in the same issue.

- [X] One
- [ ] Two

Reply:
Hmm. We don't have an issue for XPath. Also, if someone finds a rare but complex expression, we want to be able to discuss that separately, without having the ONE issue reopened every time.